cookbooks/Getting-Started/getting_started.ipynb

```diff
@@ -29,6 +31,7 @@
       ]
     },
     {
+      "attachments": {},
```

Comment:
Why is this updated? Maybe just the notebook editor?

Reply:
Ya, I changed the comment on setting .env variable and saved and it auto-added attachments.

Same for when I executed the function calling one it set language_info to some weird metadata (will remove with vim)